RIOT-OS/RIOT
# tests/posix_sleep: Kconfig uses rtt if possible

### Contribution description

Matches kconfig usemodues from tests/posix_sleep with ones generated from makefile since the test isn't enforced.

### Testing procedure

Apply:
```diff
diff --git a/.murdock b/.murdock
index acb1356065..7200f309e2 100755
--- a/.murdock
+++ b/.murdock
@@ -23,7 +23,7 @@ tests/driver_motor_driver tests/driver_mpl3115a2 tests/driver_mpu9x50
 tests/driver_mq3 tests/driver_my9221 tests/driver_nvram_spi tests/mtd_flashpage
 tests/mtd_mapper tests/driver_o* tests/driver_p* tests/driver_q*
 tests/driver_r* tests/driver_s* tests/driver_t* tests/driver_u*
-tests/driver_v*"}
+tests/driver_v* tests/posix_sleep"}
 : ${TEST_KCONFIG_native:="examples/hello-world
 tests/cb_mux*
 tests/congure_*
```

and run
```
/bin/bash -c "source .murdock; JOBS=4 compile tests/posix_sleep samr21-xpro:gnu"
```

on master and on this pr and it should show this fixes it.

### Issues/PRs references
